tools/fasta_filter_by_length/fasta_filter_by_length.xml

```diff
@@ -22,6 +22,12 @@
 			<param name="max_length" value="0" />
 			<output name="output" file="fasta_tool_filter_length_1.out" />
 		</test>
+		<test>
+			<param name="input" value="4.fasta" />
+			<param name="min_length" value="0" />
+			<param name="max_length" value="60" />
```

Comment:
Shouldn't this be more stringent, i.e.:
```suggestion
			<param name="max_length" value="54" />
```

Reply:
Deleted my previous comment... It would be more stringent but issue would not cause the test to fail.  When the maximum length is evenly divided by the line length, issue will cause larger contigs to be filtered in. 

To use value 54, we would have to change DNA line length to 54.